README.md

```diff
@@ -26,15 +26,14 @@ Puppet module to manage OpenVPN servers and clients.
 * Debian
 * CentOS
 * RedHat
-* Amazon
 
 ## Dependencies
   - [puppetlabs-concat 3.0.0+](https://github.com/puppetlabs/puppetlabs-concat)
   - [puppetlabs-stdlib 4.25.0+](https://github.com/puppetlabs/puppetlabs-stdlib)
 
 ## Puppet
 
-* Version >= 4.7.1
+* Version >= 4.25.0
```

Comment:
??

Reply:
I updated the README and saw this is not uptodate with metadata.json